packages/sources/src/icons/chevron-right.scss

```diff
@@ -4,6 +4,14 @@ chevron-right {
   color: $grasgroen;
 }
 
+chevron-right:scampi {
```

Comment:
Ik zie waar deze variant wordt _gebruikt_, maar het lukt mij niet om het te tonen in de browser

Reply:
Mij ook alleen maar in de developer toolbar, hoe zou jij dit anders oplossen?

Reply:
Ik denk dat er wat styling niet klopte, want nu werkt hte bij mij wel